ui/src/serverIO.js

```diff
@@ -266,6 +267,11 @@ class ServerIO {
 
       if (record.Signal === 'DisableSample') {
         this.dispatch(setSampleAttribute([record.sampleID], 'checked', false));
+      } else if (record.Signal === 'update') {
+        const state = store.getState();
+        if (!state.login.user.inControl) {
+          this.dispatch(fetchQueue());
```

Comment:
why not sending the queue with the emit? feels awkward to notify the UI about an update, but let the UI itself fetch the newest data

Reply:
I was thinking the same, but then I thought that it would be better to do it the same way all the time, meaning that we always fetch the queue via a http request, and we reuse the route we have in place, otherwise its like we are getting the queue in two different ways. If it makes sense

Reply:
ah yes, you have a point there